text/0096-auto-batching.md

```diff
@@ -40,41 +40,18 @@ The batch identifiers are unique and strictly increasing.
 
 #### 3.1.2. Impacts on `task` API resource
 
-- The different tasks grouped in a batch are processed within the same transaction. If a task fails within a batch, the whole batch fails.
-- A `batchUid` field is only added on fully-qualified `task` API objects. `batchUid` values are unique and strictly increasing.
+- The different tasks grouped in a batch are processed within the same transaction. But if a task fails within a batch, the whole batch does not fail, only the related task.
```

Comment:
@Kerollmops if there is any new type of error created (we would need to add [here in the docs](https://docs.meilisearch.com/reference/api/error_codes.html) and [here in the spec](https://github.com/meilisearch/specifications/blob/main/text/0061-error-format-and-definitions.md)), can you let me know, because I haven't updated the spec accordingly since I don't know what changed regarding the errors?

Reply:
It doesn't seem that any error code has been changed or added.
Below is the message returned by the failed task processed in a partially valid batch:
```json
"error": {
        "message": "Document doesn't have a `id` attribute: `{\"title\":\"lol\",\"author\":\"guy\",\"new_field\":\"bruh\"}`.",
        "code": "missing_document_id",
        "type": "invalid_request",
        "link": "https://docs.meilisearch.com/errors#missing_document_id"
}
```